src/Illuminate/Contracts/Mail/Mailer.php

```diff
@@ -21,7 +21,7 @@ public function to($users);
     public function bcc($users);
 
     /**
-     * Send a new message when only a raw text part.
+     * Send a new message with only a raw text part.
```

Comment:
I think `Send a new message only with a raw text part.` is much better.

Reply:
I was in doubt first, but I think your're right. Let's do this.

Reply:
It's best that you also do this for the rest of the docblocks then to keep consistency.

Reply:
No, reconsidering once again. It's ok as it is, see [Google Translater](https://translate.google.com/#de/en/Eine%20Nachricht%20nur%20mit%20einem%20Text.) e.g.